rust/connlib/tun/src/unix.rs

```diff
@@ -79,8 +80,12 @@ where
                             break;
                         };
                     }
+                    Err(e) if e.root_cause().is::<ip_packet::Fragmented>() => {
+                        tracing::debug!("{e:#}"); // Log on debug to be less noisy.
```

Comment:
Alternatively to this, we could also continue logging it on WARN but it might just slowly eat into our Sentry budget. What do you think @jamilbk ?